AppCenter/AppCenter/Model/MSDevice.m

```diff
@@ -132,14 +116,14 @@ - (instancetype)initWithCoder:(NSCoder *)coder {
 
 - (void)encodeWithCoder:(NSCoder *)coder {
   [super encodeWithCoder:coder];
-  [coder encodeObject:self.sdkName forKey:kMSSdkName];
-  [coder encodeObject:self.sdkVersion forKey:kMSSdkVersion];
+  [coder encodeObject:self.sdkName forKey:kMSSDKName];
+  [coder encodeObject:self.sdkVersion forKey:kMSSDKVersion];
   [coder encodeObject:self.model forKey:kMSModel];
-  [coder encodeObject:self.oemName forKey:kMSOemName];
-  [coder encodeObject:self.osName forKey:kMSOsName];
-  [coder encodeObject:self.osVersion forKey:kMSOsVersion];
-  [coder encodeObject:self.osBuild forKey:kMSOsBuild];
-  [coder encodeObject:self.osApiLevel forKey:kMSOsApiLevel];
+  [coder encodeObject:self.oemName forKey:kMSOEMName];
+  [coder encodeObject:self.osName forKey:kMSACOSName];
+  [coder encodeObject:self.osVersion forKey:kMSOSVersion];
+  [coder encodeObject:self.osBuild forKey:kMSOSBuild];
+  [coder encodeObject:self.osApiLevel forKey:kMSOSApiLevel];
```

Comment:
I'm not sure why all of these have changed, but in the new style should this be `kMSOSAPILevel`? (which i think is hard to read, btw)

Reply:
That's the convention based on Apple habits, acronyms are upper letters. Good catch.